docs/ide-usage.md

```diff
@@ -152,19 +131,16 @@ If you do not wish to use the automated setup, you can follow the [manual setup]
    1. Check `Use auto-import` and uncheck `Create separate module per source set`.
    1. Ensure `Create directories for empty content root automatically` is unchecked.
    1. Ensure `Use default gradle wrapper` is selected.
-   1. Ensure for `Gradle JVM:` that a JDK 7 with a name of `1.7` is selected.
+   1. Ensure for `Gradle JVM:` that a JDK 8 with a name of `1.8` is selected.
    1. Click `Finish`.
    1. Create a `build` folder in your project root if it does not exist while waiting for IntelliJ to finish indexing.
    1. If you used a different project root directory other than `teammates`, you will receive a dialog box with the
       message `The modules below are not imported from Gradle anymore. Check those to be removed from the ide project too:`.\
       You should see a module with your project root directory name being checked. Click `OK`.
    1. Go to `File → Project Structure... → Modules`.
       Click on the `teammates` module, then under `Sources`, click on the `build` folder and click `Excluded` and then `OK`.
```

Comment:
I'm kind of at a loss here. When is the build folder `build/`, `out/`, or `build-idea/`?

Reply:
IntelliJ uses `out` natively but when it executes gradle tasks, those tasks write to the gradle property `projectDir` as defined in the gradle script. Hence, both directories are used. There's no easy way to change the default folder to something else other than `out` via gradle script, but I managed to workaround this by setting `projectDir` to a location nested within `out`. Now everything goes into `out` :)